src/pages/home/report/ReportActionCompose.js

```diff
@@ -441,15 +463,18 @@ class ReportActionCompose extends React.Component {
     }
 
     /**
-     * Clean data related to EmojiSuggestions
+     * Clean data related to EmojiSuggestions and MentionSuggestions
      */
-    resetSuggestedEmojis() {
+    resetSuggestions() {
         this.setState({
             suggestedEmojis: [],
+            suggestedMentions: [],
             highlightedEmojiIndex: 0,
             colonIndex: -1,
-            shouldShowSuggestionMenu: false,
-            isEmojiPickerLarge: true,
+            atSignIndex: -1,
+            shouldShowEmojiSuggestionMenu: false,
+            shouldShowMentionSuggestionMenu: false,
+            isAutoSuggestionPickerLarge: true,
```

Comment:
This is `false` when we initially set the state. Why is it `true` here?

Reply:
That part was exactly the same in the original implementation of the emoji suggestions, I'm not sure of exact reasoning behind it (nor if it makes any difference, as it will be recalculated when the `calculateEmojiSuggestions` will be fired)